vllm/sequence.py

```diff
@@ -971,6 +999,8 @@ class SamplerOutput:
 
     # On-device tensor containing the sampled token ids.
     sampled_token_ids: Optional[torch.Tensor] = None
+    # sampled_token_ids_numpy: Optional[List[int]] = None
+    sampled_token_ids_numpy: Optional[numpy.ndarray] = None
```

Comment:
Do we have to keep both `sampled_token_ids` and `sampled_token_ids_numpy`? Can we only keep the numpy version? Or is the change too big for this PR?

Reply:
It would be too big for this PR as `sampled_token_ids` is used everywhere. @youkaichao was having a PR to deal with these but that PR is idle for a while. We should change all dat a types in SequenceGroup and SequenceOutput in that PR.

Reply:
now named `sampled_token_ids_cpu` as we don't really need to convert to numpy